package/src/RNCSliderNativeComponent.web.tsx

```diff
@@ -145,18 +145,36 @@ const RCTSliderWebComponent = React.forwardRef(
       }
     }, [initialValue, updateValue, animatedValue]);
 
-    const onResize = () => {
-      hasBeenResized.current = true;
-    };
     React.useEffect(() => {
+      const invalidateContainerPosition = () => {
+        containerPositionInvalidated.current = true;
+      };
+      const onDocumentScroll = (e: Event) => {
+        if (
+          // Avoid all other checks if already invalidated
+          !containerPositionInvalidated.current &&
+          containerRef.current &&
+          // @ts-ignore
+          e.target.contains(containerRef.current)
+        ) {
+          invalidateContainerPosition();
+        }
+      };
       //@ts-ignore
-      window.addEventListener('resize', onResize);
+      window.addEventListener('resize', invalidateContainerPosition);
```

Comment:
@motiz88 you should probably subscribe to resize events the react-native way:

https://necolas.github.io/react-native-web/docs/use-window-dimensions/

Reply:
@jspizziri
1. This use of the `resize` event was pre-existing and I had no intention of changing it as part of this bugfix. I merely renamed `onResize` to `invalidateContainerPosition` to better communicate what it does (and reuse it in the `onScroll` handler).
2. IMO, given this is platform-specific code, use of DOM primitives is completely fine. It's also not directly interchangeable with `useWindowDimensions` (which would tie us to the React component lifecycle in a way that might not be the most correct/efficient for this use case)